README.md

```diff
@@ -37,6 +37,42 @@ or add
 
 to the `require` section of your `composer.json` file.
 
+## Usage with Yii Console
+
+Just install `yiisoft/yii-console` package and you are ready to go.
+
+## Usage with Symfony Console
```

Comment:
For use with Symfony Console we should add commands to Symfony Console and do not use files from bin.

Reply:
https://symfony.com/doc/current/components/console.html#creating-a-console-application Well in the documents it clearly says that you must register a script and add the commands, it is what is done in this PR.

Reply:
We should add instruction for use commands with symfony console that alreasy used in user app.

Reply:
You have to investigate it, i suggest merging this PR, and do it in another.